<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="0" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-06-03T15:37:16 EET" name="Suite" finished-at="2023-06-03T15:37:31 EET" duration-ms="14722">
    <groups>
    </groups>
    <test started-at="2023-06-03T15:37:16 EET" name="Test" finished-at="2023-06-03T15:37:31 EET" duration-ms="14722">
      <class name="tests.LogInTestCases">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:tests.LogInTestCases@21282ed8]" started-at="2023-06-03T15:37:16 EET" name="setup" finished-at="2023-06-03T15:37:19 EET" duration-ms="3079" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="invalidLogIn_wrongCredintials()[pri:0, instance:tests.LogInTestCases@21282ed8]" started-at="2023-06-03T15:37:19 EET" name="invalidLogIn_wrongCredintials" finished-at="2023-06-03T15:37:23 EET" duration-ms="3586" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidLogIn_wrongCredintials -->
        <test-method is-config="true" signature="tear_down()[pri:0, instance:tests.LogInTestCases@21282ed8]" started-at="2023-06-03T15:37:23 EET" name="tear_down" finished-at="2023-06-03T15:37:23 EET" duration-ms="520" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tear_down -->
        <test-method signature="invalid_logins_excel(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:tests.LogInTestCases@21282ed8]" started-at="2023-06-03T15:37:24 EET" name="invalid_logins_excel" data-provider="dataexcelsheet" finished-at="2023-06-03T15:37:24 EET" duration-ms="0" status="SKIP">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "org.apache.poi.xssf.usermodel.XSSFCell.getStringCellValue()" because the return value of "org.apache.poi.xssf.usermodel.XSSFRow.getCell(int)" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException: Cannot invoke "org.apache.poi.xssf.usermodel.XSSFCell.getStringCellValue()" because the return value of "org.apache.poi.xssf.usermodel.XSSFRow.getCell(int)" is null
at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:50)
at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:146)
at org.testng.internal.Parameters.handleParameters(Parameters.java:798)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:791)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.NullPointerException: Cannot invoke "org.apache.poi.xssf.usermodel.XSSFCell.getStringCellValue()" because the return value of "org.apache.poi.xssf.usermodel.XSSFRow.getCell(int)" is null
at util.GetData.read_excel_sheet1(GetData.java:31)
at tests.LogInTestCases.dataexcelsheet(LogInTestCases.java:134)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:77)
at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:46)
... 25 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalid_logins_excel -->
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:tests.LogInTestCases@21282ed8]" started-at="2023-06-03T15:37:24 EET" name="setup" finished-at="2023-06-03T15:37:26 EET" duration-ms="1967" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="validLogIn()[pri:0, instance:tests.LogInTestCases@21282ed8]" started-at="2023-06-03T15:37:26 EET" name="validLogIn" finished-at="2023-06-03T15:37:30 EET" duration-ms="4705" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validLogIn -->
        <test-method is-config="true" signature="tear_down()[pri:0, instance:tests.LogInTestCases@21282ed8]" started-at="2023-06-03T15:37:30 EET" name="tear_down" finished-at="2023-06-03T15:37:31 EET" duration-ms="500" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tear_down -->
      </class> <!-- tests.LogInTestCases -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
